### PR TITLE
RS-2 Create project directories

### PR DESCRIPTION
**Pull request by silaghiandrei**

### CODE REVIEW

File: Ride_Sharing_UI/src/test.txt
Comment (BogdanSavianu) [inline Ride_Sharing_UI/src/test.txt:1]: asta ca sa nu fie director gol?

**Review by BogdanSavianu — COMMENTED** [review]
CrudRepository nu era interfata si clasele din implementation sa o implementeze?
La fel la service alea nu erau interfete si in implementation sa fie clasele ce le implementeaza? Also putem face interfata generica GenericService<T> si DriverService, OrderService etc sa o extinda

**Comment by LoredanaBln** [comment]
it's a good practice to squash your commits, so it's easier for us to rebase main and keep a cleaner history.

**Comment by BogdanSavianu** [comment]
You could add DriverRepositoryImplementation implements DriverRepository at this stage too and same for the other implementations and get these out of the way.

**Comment by BogdanSavianu** [comment]
LGTM

**Comment by silaghiandrei** [comment]
Resolved the issues

**Pull request merged**